docs/sources/developers/plugins/metadata.md

```diff
@@ -21,22 +21,28 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
 | `module`        | string                  | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
 | `partials`      | [object](#partials)     | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `preload`       | boolean                 | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
 | `queryOptions`  | [object](#queryoptions) | No       | For data source plugins. There is a query options section in the plugin's query editor and these options can be turned on if needed.                                                                                                                                                                                                                                                                    |
 | `routes`        | [object](#routes)[]     | No       | For data source plugins. Proxy routes used for plugin authentication and adding headers to HTTP requests made by the plugin. For more information, refer to [Authentication for data source plugins](https://grafana.com/docs/grafana/latest/developers/plugins/authentication/).                                                                                                                       |
+| `signature`     | string                  | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Not configured externally -- this is filled inat runtime depending on the state of the signature